README.md

```diff
@@ -30,10 +30,16 @@ You'll also need to clone a new `.env` file from the `.env.tempalate` to store l
 $ cp .env.template .env  # (first time only)
 ```
 
-The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
+The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). 
+## In the .env file you will need to put in your trello keys, if you do not already have a trello account go to trello.com and register
+## In order for the application to work you should setup a My TODO board with three cards called Doing, DoneList and ToDoList
+## These are currently hardcoded in trello_base.py for this demo.
```

Comment:
The formatting is wrong... You can view the file if you go to https://github.com/sixela70/DevOps-Course-Starter/tree/exercise-2 and scroll down.

Or in VSCode you can also view .md files with formatting by clicking an icon to the top right of the file

And the lines below, listing the env vars, should probably be next to the instruction `In the .env file you will need to...`. At the moment it's kind of confusing.